vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 
 cd oms-docker/docker
-docker-compose up -d
-echo "############# ok until here"
-docker-compose logs -f omscore-bootstrap
-docker-compose logs -f omsevents-bootstrap
-echo "############# done"
+make bootstrap
```

Comment:
Actually another thing is the filesystem drivers: does not depend on the docker daemon but on the OS.. sometimes they can bitch (I think that's the issue that my company has gotten into)